dashboard/src/components/AppView/AppView.tsx

```diff
@@ -78,7 +78,7 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
     if (!newApp) {
       return;
     }
-    let manifest: IResource[] = yaml.safeLoadAll(newApp.manifest);
+    let manifest: IResource[] = yaml.loadAll(newApp.manifest, undefined, { json: true });
```

Comment:
Can you add a TODO statement about the upgrade? Thanks!

Reply:
done thanks